pwiz_tools/Skyline/Controls/Graphs/GraphSpectrum.cs

```diff
@@ -732,6 +732,9 @@ private SpectrumGraphItem MakeGraphItem(SpectrumDisplayInfo spectrum, SpectrumNo
                 rankAdducts,
                 rankTypes,
                 null);
+
+            var isHighRes = !Equals(settings.TransitionSettings.FullScan.ProductMassAnalyzer,
```

Comment:
GraphSpectrum is used for displaying spectra from a library. The spectra in the library were often acquired on a completely different instrument than the raw files in the current document, so you should not be looking at TransitionSettings to decide whether the library spectrum is high resolution or not.

I think the spectrum should always display the mass error, even if the spectrum is low resolution. Skyline does not calculate the mass error when extracting chromatograms for low resolution data, but if the user has asked to see the mass error in this graph, Skyline should show it.

Reply:
I removed the high res logic to show mass error for all types of spectra.

Reply:
I don't know. Should it show mass error in PPM for low-resolution spectra? I would have thought low resolution should show mass error in m/z units. I guess we can leave it in PPM and wait to see what users think.